### PR TITLE
Coachella 2023

### PR DESCRIPTION
**Pull request by dacuna85**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by dacuna85** [comment]
https://github.com/instructlab/taxonomy/issues/832

File: knowledge/music/events/cochella_2023/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/music/events/cochella_2023/qna.yaml:1]: Please add your GitHub Username here.

File: knowledge/music/events/cochella_2023/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: music
+seed_examples:
+- answer: a music festival that took place over two weekends in mid-April 2023
+  question: What is coachella?
+- answer: The festival was headlined by Puerto Rican rapper Bad Bunny, South Korean girl group Blackpink, American singer Frank Ocean, and American rock band Blink-182.[a]
+  question: who headlined coachella_2023?
+- answer: Ocean's Coachella performance was harshly criticized from audience members, critics, and fans.[5] The Festive Owl, speaking to several sources, later stated that a group of people surrounding Ocean were intended to be ice skaters and had practiced for weeks
```
Comment (jjasghar) [inline knowledge/music/events/cochella_2023/qna.yaml:8]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please make these fixes then we can run the linter.

**Pull request closed without merging**